tasks/Open To Evacuate or Not to Evacuate.md

```diff
@@ -0,0 +1,44 @@
+3. After answering the survey, each subject was asked to take a tutorial before the actual game would begin. In the tutorial, each subject separately interacted with three dummy players in a 45-second practice game.
+4. After the practice game, subjects were assessed for their comprehension of the game rules and payment structure using three multiple-choice questions, each with three options. If they failed to select the correct answer to any of the questions, they were dropped from the game. 
+5. At 720 s after the tutorial began, a ‘Ready’ button became visible simultaneously to all the subjects who completed the tutorial and passed the comprehension tests. 
+6. The real games started 30 s after the ‘Ready’ button showed up. If subjects did not click the button before the game started, they were dropped. 
+7. The games of networked groups required an exact number of subjects. When the subjects who successfully clicked the button were more than the required number, surplus subjects, who were randomly selected, were dropped from the game. When the number of qualified subjects was less than the required number, the game did not start (and subjects got their show-up fee). 
+8. At the start of the game, we selected one subject (the ‘informant’) at random who was informed in advance whether a disaster would indeed strike or not. The behaviours of the informants did not vary significantly across the network treatments
+>  The exact sentence that the informants received in their game screen was ‘A disaster is going to strike!’ when a disaster would strike or ‘There is no disaster’. when a disaster would not strike. Uninformed subjects received notification that ‘A disaster may or may not strike’.
+9. The other subjects were informed that some players had accurate information about the disaster, and they were also told that immediate neighbours of the informant would know the identity of the informant by our marking ‘i’ on the informant's node in their network diagram in the game screen. When a disaster struck before they evacuated, the subjects earned no bonus. Otherwise, they earned a bonus of $2.00 without the disaster or $1.00 with the disaster by spending $1.00 to evacuate, plus $0.10 per other player who took the correct action accordingly 
+10. Each session ended in 75 s. In half of the sessions, a disaster struck at the end of the game. We did not inform any subjects, including the informants, when their sessions would end, the global network structure in which they were embedded, or how many informants were in the game.
+
+## Roles 
+The Informant: informed in advance whether a disaster would indeed strike or not
+
+## Instructions
+Reached out to instructors 5/5
+
+# Criteria
+## Performance calculation
+A formal specification of how performance is evaluated based on the specific materials and the teams actions in a given trial.  
```

Comment:
Performance was calculated based on the participant's ability to correctly avoid an impending disaster, and the payoffs are displayed in the Stimuli sections